README.rst

```diff
@@ -38,6 +38,11 @@ All of these features are on by default but can be disabled with flags.
   test run, but you can pass in ``--randomly-seed`` to repeat a
   randomness-induced failure.
 
+* If
+  `model bakery <https://model-bakery.readthedocs.io/en/latest/>`_
+  is installed, its random state is reset at the start of every test. This
+  allows for repeatable use of its random fixture field values.
```

Comment:
The list of packages is alphabetized, move to line 58.

Reply:
Alphabetizing keeps slipping my mind haha. Thanks for pushing the updates.